docs/source/how-to/index.rst

```diff
@@ -156,7 +156,7 @@ For more complex scenarios, you can create fully customize workflows where you c
 
       Learn how to split a model into multiple components.
 
-      :octicon:`arrow-right;1em;sd-text-info` `Model splitting <model-splitting.html>`_
+      :octicon:`arrow-right;1em;sd-text-info` `Model splitting <configure-workflows/model-splitting.html>`_
```

Comment:
btw, do you know if it's possible to reference to the corresponding .md file instead of the generated .html files like in https://github.com/microsoft/Olive/blob/main/docs/source/index.md?plain=1?

Reply:
No it doesn't work. This grid-item-card seems only can work with .html. If using aabb.md, it will redirect to aabb.md, while aabb.md doesn't exist only aabb.html exists.